terraform/common.tf

```diff
@@ -71,3 +71,16 @@ variable "cortex_instance_endpoint" {
   # change to your cortex endpoint
   default = "https://aps-workspaces-gamma.us-west-2.amazonaws.com/workspaces/ws-31eb305d-3208-42d5-a7f4-32ce1191e699"
 }
+
+variable "ssm_mode" {
```

Comment:
maybe change it to `install_mode` with some 'enums' like package manager (i.e. download deb/rpm directly in user data) or `ssm`

Or reuse `install_package_source` (See comments in other files).

Reply:
Agree. I will use install_package_source for installation methods. However I want to keep ssm_mode to disable terraform resources for SSM testcase.